yak_ros/CMakeLists.txt

```diff
@@ -150,13 +150,20 @@ endif()
 if ($ENV{ROS_VERSION} VERSION_EQUAL "1")
     set(ROS_LIB_DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION})
     set(ROS_SHARE_DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
+
+    install(TARGETS ${PROJECT_NAME}_node ${PROJECT_NAME}_visualizer_ros1
+      ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
+      LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
+      RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+    )
 else()
     set(ROS_LIB_DESTINATION lib/${PROJECT_NAME})
     set(ROS_SHARE_DESTINATION share/${PROJECT_NAME})
+
+    install(TARGETS ${PROJECT_NAME}_node
+            RUNTIME DESTINATION ${ROS_LIB_DESTINATION})
 endif()
```

Comment:
based on what was being done before it may be preferred to create a PACKAGE_LIBRARIES list then in the ROS1 if append the visualizer target and change the original install to.

```
install(TARGETS ${PACKAGE_LIBRARIES}
        RUNTIME DESTINATION ${ROS_LIB_DESTINATION})
```

Reply:
There is only one executable and one library, I think it would probably be overkill to create a list of the targets.  Also, the format of just installing into the runtime destination is not sufficient when other ROS1 packages are trying to link against and run this code, which is why I introduced the full standard ROS1 install rule.

Reply:
These changes highlight some other things we'll need to change, namely that there's a bunch of leftover CMake logic from when this was trying to be a dual ROS1/ROS2 package (which has been preempted by [yak_ros2](https://github.com/ros-industrial/yak_ros2/)).

I think these changes are essentially OK for now.